adoc/reference-webui-systems.adoc

```diff
@@ -1516,6 +1517,17 @@ The new virtual machine will start immediately after it has been defined.
 
 // TODO Add a screenshot of the create page showing the additional fields
 
+[float]
+[[sdc-virt-host-guest-graphical-console]]
+===== Display a virtual machine graphical console[Salt]
+
+When showing virtual machine graphical console, the user may be prompted a password.
```

Comment:
``The virtual machine graphical console might prompt you for a password.``

Reply:
Fixed.